default/stringtables/en.txt

```diff
@@ -1638,6 +1638,12 @@ Sets the number of turns between autosaves.
 OPTIONS_DB_AUTOSAVE_LIMIT
 Sets the maximum number of autosave files to keep.
 
+OPTIONS_DB_AUTOSAVE_INTIAL_TURN
+If true, autosave the initial turn before any game play.
```

Comment:
It is slightly clearer, but the "if true, stuff" idiom is just too awkward to actually use, in my opinion...  At least switch to "Enables an autosave on the initial turn..." or similar.

Reply:
I changed the wording to "Enable single-player autosaves." etc.